includes/transformer/class-base.php

```diff
@@ -90,7 +90,7 @@ protected function transform_object_properties( $activity_object ) {
 			if ( \method_exists( $this, $getter ) ) {
 				$value = \call_user_func( array( $this, $getter ) );
 
-				if ( ! empty( $value ) ) {
+				if ( ! empty( $value ) || false === $value ) {
```

Comment:
```suggestion
				if ( isset( $value ) ) {
```

Reply:
I went with an explicit `null` check, if that's okay

Reply:
Just to be explicit. I wouldn't expect there to be a situation where `$value` is truly not set after calling `call_user_func()`.

Reply:
but then we would at least have exactly the same behaviour everywhere (to_array is using isset)... but I do not want to be pedantic...